build.gradle

```diff
@@ -8,10 +8,11 @@ buildscript {
 
     ext {
         kotlin_version = '1.2.51'
-        android_plugin = '3.1.2'
+        android_plugin = '3.1.4'
         ktlint_version = '0.24.0'
 
-        support_library_version = '28.0.0-alpha3'
+        support_library_version = '27.1.1'
```

Comment:
Revert gradle changes.

Reply:
Actually I was unable to make it work with this version of library (28.0.0-alpha3).. getting 
```
Unable to resolve dependency for ':app@debug/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@debug/compileClasspath': Could not resolve com.android.support:preference-v7:28.0.0-alpha3.

Unable to resolve dependency for ':app@debug/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugAndroidTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugAndroidTest/compileClasspath': Could not resolve com.android.support:preference-v7:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugAndroidTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugUnitTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugUnitTest/compileClasspath': Could not resolve com.android.support:preference-v7:28.0.0-alpha3.

Unable to resolve dependency for ':app@debugUnitTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@release/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@release/compileClasspath': Could not resolve com.android.support:preference-v7:28.0.0-alpha3.

Unable to resolve dependency for ':app@release/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@releaseUnitTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.

Unable to resolve dependency for ':app@releaseUnitTest/compileClasspath': Could not resolve com.android.support:preference-v7:28.0.0-alpha3.

Unable to resolve dependency for ':app@releaseUnitTest/compileClasspath': Could not resolve com.android.support:design:28.0.0-alpha3.
```

I suppose it may be related to other local changes I had to make such as [wrong observer](https://github.com/CypherpunkArmory/UserLAnd/pull/237#discussion-diff-215704153R69) and [SuppresLint annotation](https://github.com/CypherpunkArmory/UserLAnd/pull/237#discussion-diff-215704201R12) , following advice [here](https://stackoverflow.com/questions/50736578/after-updating-android-studio-to-3-1-2-i-get-failed-to-load-appcompat-actionb) did not help. I will look into this further. As mentioned in PR description there are the changes that I will revert once i will solve them. Sorry in advance if they will occur in PR sketches.